health-services/project-factory/src/server/utils/sheetManageUtils.ts

```
@@ -0,0 +1,356 @@
+import { v4 as uuidV4 } from "uuid";
+import config from "../config";
+import { produceModifiedMessages } from "../kafka/Producer";
+import * as ExcelJS from "exceljs";
+import { callMdmsSchema, createAndUploadFileWithOutRequest } from "../api/genericApis";
+import { getLocalizedHeaders, getLocalizedMessagesHandlerViaLocale, handledropdownthings } from "./genericUtils";
+import { getLocalisationModuleName } from "./localisationUtils";
+import { getLocalizedName } from "./campaignUtils";
+import {  adjustRowHeight, enrichTemplateMetaData, freezeUnfreezeColumns, manageMultiSelect, updateFontNameToRoboto } from "./excelUtils";
+import * as path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)